javascript/stemwords-es6.mjs

```diff
@@ -0,0 +1,129 @@
+import fs from 'fs';
```

Comment:
It would be more helpful for ongoing maintenance if this was written such that the parts which are effectively the same as the existing version were actually the same.  (I don't mind if we adjust the existing code if that makes that easier, so long as it doesn't stop it working in situations where it currently works.)

Even ignoring whitespace, the current diff looks more like diffing two unrelated files.